index.html

```diff
@@ -1,26 +1,31 @@
-jessica Baker
-
-"About"
-My name is Jessica Baker. I am 30 years old. I live in Palatine Il
-with my 3 chrildren. I am a student at CTD school.My favorite subject
-is math. My hobbies are listening to music and taking my chrildren 
-out to fun places.
-
-"Experience"
--javascript
--linux
-
-"Connect"
-https://github.com/Jessica12492/intro-to-programming-dorado
-https://mail.google.com
-https://www.facebook.com
-
-
-
-
-
-
-
-
-
-
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Personal Portfolio Website</title>
+  <meta charset="UTF-8">
+</head>
+
+<body>
+  <section>Jessica Baker<h2>About</h2>My name is Jessica Baker. I am 30 years old. I live in Palatine il with my 3
+    children. I am a student at CTD school. My favorite subject is math. My hobbies are listening to music , shopping
+    and taking my children out to fun places.</section>
+  <section>
+    <h2>Experience</h2>
+    <ul>
+      <li>javascript</li>
+      <li>linux </li>
+    </ul>
+  </section>
+  <section>
+    <h3>Connect</h3>
+    <ul>
+      <li> https://github.com/Jessica12492/intro-to-programming-dorado</li>
+      <li> https://mail.google.com</li>
+      <li> https://www.facebook.com</li>
+    </ul>
+  </section>
+  <img src="../myself.jpg" alt="Picture of Me" width="100" height="90">
```

Comment:
Nice! great use of additional HTML tags like \*\*img\*\*. Just so you know, in this case this won't actually show any image; since there's no file called \`myself.jpg\` in this directory/repo. 

If you wanted to add an image yourself you could add it to the same \`intro-to-programming-dorado\` directory, and in this tag, you would updated the \`src="../myself.jpg"\` to \`src="./\<name\_of\_image\_file\_with\_extension"\` (notice just using the single dot instead of two dots, since the image will be in the same folder as \`index.html\`).

You could also just use an image from anywhere online if you prefer. So if I wanted to for example show \[this picture\](https://media.istockphoto.com/id/992637094/photo/british-short-hair-cat-and-golden-retriever.jpg?s=612x612&w=0&k=20&c=MPxqqfyY1mhqlrqD9xCC3NA14qSR0DfZnD5SVJxs2Ik=) of a cute cat-and-dog pair on my website, I would write something like:

\`\`\`  
\<img   
 src="https://st2.depositphotos.com/1606449/8228/i/950/depositphotos_82289876-stock-photo-happy-dog-and-cat.jpg"  
 width="100"  
 height="100"  
/>  
\`\`\`

You'll notice that \`img\` is one of the tags that's considered "self-closing". So you don't need to do opening tag, then content, then closing tag (eg: \`\<img>something\</img>\`) instead, you just do the slash at the end of the opening tag (\`\<img />\`)

Reply:
I thought the directory was my linux file. because that's how I check to see how my file looks. like when I click on files, then I  click on linux files, and my intro to programming files to work on are there and saved. so i moved my picture in the linux files where everything is. I save it as myself. 
![image](https://user-images.githubusercontent.com/116823772/201747140-843a3dfa-66a6-42ee-b08c-6f62863b299f.png)

Reply:
when i changed it to `src="./<name_of_image_file_with_extension"` the image doesn't show for me

Reply:
oh  i get it. i changed it and added it to git

Reply:
i did the commands. can you see the changes?